src/Microsoft.Azure.SignalR.Common/DefaultServiceEventHandler.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+
+using Microsoft.Azure.SignalR.Protocol;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.SignalR
+{
+    public class DefaultServiceEventHandler : IServiceEventHandler
+    {
+        private readonly ILogger _logger;
+
+        public DefaultServiceEventHandler(ILoggerFactory loggerFactory)
```

Comment:
DI `ILogger<DefaultServiceEventHandler>` directly?

Reply:
failed to run unit test if using `ILogger<DefaultServiceEventHandler>`